classes/class-p4-search.php

```diff
@@ -399,39 +491,37 @@ protected function set_results_context( &$context ) {
 						if ( 'act' === basename( get_permalink( $post->post_parent ) ) ) {
 							$content_type_text = __( 'ACTION', 'planet4-master-theme' );
 							$content_type      = 'action';
-							$context['posts_data']['found_actions']++;
+							$context['content_types']['0']['results'] ++;
```

Comment:
Just a minor thing.. Please remove the extra space before post-increment operator.
I don't know why, but Phpstorm adds them if you use wordpress coding style.